### PR TITLE
Add state-of-tic-tac-toe exercise

### PR DESCRIPTION
**Pull request by keiravillekode**

### CODE REVIEW

File: exercises/practice/state-of-tic-tac-toe/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Instructions
+
+In this exercise, you're going to implement a program that determines the state of a [tic-tac-toe][] game.
+(_You may also know the game as "noughts and crosses" or "Xs and Os"._)
+
+The games is played on a 3×3 grid.
```
Comment (kahgoh) [inline exercises/practice/state-of-tic-tac-toe/.docs/instructions.md:6]:
```suggestion
The game is played on a 3×3 grid.
```

This suggested change is to mirror a recent update in the problem specifications to fix this grammar (exercism/problem-specifications#2482).

**Review by kahgoh — COMMENTED** [review]
This looks good! Just one minor suggested change to update the instructions.md.

File: exercises/practice/state-of-tic-tac-toe/.docs/instructions.md (hunk 1 above)
Comment (keiravillekode) [reply exercises/practice/state-of-tic-tac-toe/.docs/instructions.md]: Done

**Review by kahgoh — APPROVED** [review]
Thanks! I noticed the CI failed due to a failure in the Gigasecond tests (totally unrelated to these changes, so happy to approve and merge). I'm planning to have a look and raise a PR to fix the CI.

**Pull request merged**